.nf-core.yml

```diff
@@ -1,9 +1,11 @@
-bump_version: null
 lint:
   files_exist:
     - assets/nf-core-meta-val_logo_light.png
+    - assets/nf-core-metaval_logo_light.png
     - docs/images/nf-core-meta-val_logo_light.png
     - docs/images/nf-core-meta-val_logo_dark.png
+    - docs/images/nf-core-metaval_logo_light.png
+    - docs/images/nf-core-metaval_logo_dark.png
```

Comment:
how many versions of the logo do you have?

Reply:
Mainly thinking about the `meta-val` vs `metaval`

Reply:
I’m keeping `meta-val` for now, but I’m confused between them. In practice it’s not good to have a dash in the repo name. I’m considering renaming `meta-val` to either `metaeval` or `medieval`, as suggested by @jfy133.

Reply:
I'd just go for consistency, you have some `metaval` in your codebase, and some `meta-val` mixed in

Reply:
I am going to merge this PR and try to fix the consistency of pipeline name in the separate PR.